tmp_scripts/make_allvars_files.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
```

Comment:
What are these two scripts about? Something temporary needed while the PR is created and then removed at the last moment?

If so that's an interesting strategy that sounds useful.